libflux/src/semantic/types.rs

```diff
@@ -119,14 +119,28 @@ impl fmt::Display for MonoType {
 // Tvar stands for type variable.
 // A type variable holds an unknown type.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
-pub struct Tvar(i64);
+pub struct Tvar(pub i64);
 
 impl fmt::Display for Tvar {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         write!(f, "t{}", self.0)
     }
 }
 
+// Fresher returns a fresh type variable with incrementing id.
+pub struct Fresher(i64);
```

Comment:
yep, it needs to because it is used in `analyze`.

I could move it to analyze and make it private because I guess it wouldn't be used anywhere else.

Reply:
Sorry I missed this. `analyze` will need to take a `Fresher` as input and so it should be public so that the same `Fresher` can be used when analyzing multiple packages.

Reply:
Yeah. I guessed that was the reason. I can revert the last commit then.